### ISSUE TITLE
ScanNet Train/Val/Test

### ISSUE THREAD

**Issue by mohammed-amr** [zju3dv/NeuralRecon#33]
Hello, 

I'm trying to make sense of the splits you use with ScanNet. Which split is used to produce Tables 1 and 2 in the paper? 

The test code in the repo seems to use ScanNet's test split (in `scans_test`), while the train code seems to use both ScanNet's train and val splits (in `scans`). 

Please clear this up for me. 

Thanks!

**Comment by JiamingSuen** [comment]
We used two different splits in Table 1 and 2, following Atlas and BA-Net. Please refer to Section 4.1 in our paper for more details.

**Comment by mohammed-amr** [comment]
Unfortunately that section doesn't answer my question; it just says evaluation is done with two different splits, pointing to ATLAS and BA-Net. 

Which split does this repo use? It looks like it uses the entirety of train/val for training, and the test set in scans_test for final evaluation.

**Comment by JiamingSuen** [comment]
This repo uses the split from Atlas. No, we only used the train split for training. Please refer to the following code:
1. data loader for the training set: https://github.com/zju3dv/NeuralRecon/blob/master/main.py#L117
2. the loader will read the training split defined in `self.mode`: https://github.com/zju3dv/NeuralRecon/blob/master/datasets/scannet.py#L31, 
3. the `fragments_train.pkl` file is generated from here: https://github.com/zju3dv/NeuralRecon/blob/master/tools/tsdf_fusion/generate_gt.py#L257

**Comment by mohammed-amr** [comment]
Awesome! Thank you!
